.github/ISSUE_TEMPLATE/new-release.md

```diff
@@ -0,0 +1,56 @@
+---
+name: New Release
+about: Propose a new release
+title: Release vX.y.z
+
+---
+
+## Release Checklist
+<!--
+Please do not remove items from the checklist
+-->
+- [ ] Network-operator related component versions in Helm chart are up to date. (otherwise, submit PR to update)
+  - [ ] node-feature-discovery
+  - [ ] SR-IOV Network Operator
+  - [ ] Manifest related component default versions
+- [ ] Example folder is up to date (otherwise submit PR to update)
+- [ ] Update network-operator Helm `Chart.yaml` with the release version (`appVersion`, `version` fields)
+  - [ ] Ensure Helm CI is passing on updated Chart.
+- [ ] Tag release
+- [ ] Create release in github
+  - [ ] Changelog for current release with: `New Features`, `Bug Fixes`, `Known Limitations`
+  - [ ] Release artifacts for current release
+  - [ ] Add link to https://mellanox.github.io/network-operator
+- [ ] Update gh-pages branch
```

Comment:
if we capture it in the issue we can remove the release.md in gh_pages

Reply:
yes i will remove it.

Reply:
will be removed in a separate PR as its a separte branch